cmd/jaeger/internal/components_test.go

```diff
@@ -38,9 +38,6 @@ func TestComponents(t *testing.T) {
 	assert.NotNil(t, factories.Exporters)
 	assert.NotNil(t, factories.Processors)
 	assert.NotNil(t, factories.Connectors)
-
-	_, jaegerReceiverFactoryExists := factories.Receivers["jaeger"]
-	assert.True(t, jaegerReceiverFactoryExists)
```

Comment:
Why is this deleted?

Reply:
good catch did remove it temporary while I was fixing the tests and forgot to add it in the last commit whit the new implementation. 

Done